src/drive/mobile/lib/tracker.js

```diff
@@ -51,7 +51,9 @@ export const startHeartBeat = () => {
   if (trackerInstance) {
     try {
       trackerInstance.push(['enableHeartBeatTimer', mobileHeartBeatDelay])
-    } catch (err) {}
+    } catch (err) {
+      //console.log(err)
```

Comment:
Why the commented out logs? Is it so the variable gets used?

Reply:
No empty blocks...

I added these lines for several reasons : 
- I think we should really take care of our errors and how we manage them. 
- I have to add several `disable case assignation` because I don't know really what to do with these cases and if we need to remove it or not